src/cljstyle/config.clj

```diff
@@ -188,18 +193,25 @@
 (defn ignored?
   "True if the file should be ignored."
   [config ^File file]
-  (some
-    (fn test-rule
-      [rule]
-      (cond
-        (string? rule)
-        (= rule (.getName file))
-
-        (pattern? rule)
-        (boolean (re-seq rule (.getCanonicalPath file)))
-
-        :else false))
-    (:file-ignore config)))
+  (or
+    (some
+      (fn test-rule
+        [rule]
+        (cond
+          (string? rule)
+          (= rule (.getName file))
+
+          (pattern? rule)
+          (boolean (re-seq rule (.getCanonicalPath file)))
+
+          :else false))
+      (:file-ignore config))
+    (some
+      (fn test-glob
+        [glob]
+        (let [path-matcher (.getPathMatcher (FileSystems/getDefault) (str "glob:" glob))]
```

Comment:
I'm open to moving this to `main`; I wasn't sure if it's better to not recreate these with each file vs not have `main` know about `FileSystems` and `PathMatcher`s.

Reply:
I don't think these are too expensive to create; I'd leave it simpler like this and let the profiling results drive changes.